src/App/MainPage.xaml

```diff
@@ -37,123 +44,128 @@
+                    <RowDefinition Height="Auto" />
+                </Grid.RowDefinitions>
+                <TextBlock x:Name="Header" x:Uid="FactoryOrchestrator" Text="" TextWrapping="Wrap" VerticalAlignment="Center" HorizontalAlignment="Left" Grid.Column="0" FontWeight="Bold" Style="{StaticResource TitleTextBlockStyle}"/>
+                <StackPanel x:Name = "MainControlPanel" Orientation="Horizontal" HorizontalAlignment="Right" Padding="40,0,0,0" Grid.Row="0" Grid.Column="1">
+                    <StackPanel x:Name="IndicatorPanel" Orientation="Horizontal" Background="{ThemeResource ControlAltFillColorTransparent}" HorizontalAlignment="Left" CornerRadius="5" Padding="1">
+                        <StackPanel Orientation="Vertical" Visibility="Collapsed" VerticalAlignment="Center" HorizontalAlignment="Left" Margin="5">
+                            <TextBlock x:Name="NetworkName" Text="" Style="{StaticResource CaptionTextBlockStyle}" VerticalAlignment="Center" TextWrapping="Wrap" IsTextSelectionEnabled="True"/>
+                            <TextBlock x:Name="NetworkIp" Text="" Style="{StaticResource CaptionTextBlockStyle}" VerticalAlignment="Center" TextWrapping="Wrap" IsTextSelectionEnabled="True"/>
+                        </StackPanel>
+                        <StackPanel Orientation="Vertical" VerticalAlignment="Center" HorizontalAlignment="Left" Margin="5">
+                            <TextBlock x:Name="OEMVersionHeader" Text="" VerticalAlignment="Center" HorizontalAlignment="Left" TextWrapping="Wrap" Style="{StaticResource CaptionTextBlockStyle}" IsTextSelectionEnabled="True"/>
+                            <TextBlock x:Name="AppVersionHeader" Text=""  VerticalAlignment="Top" HorizontalAlignment="Left" TextWrapping="Wrap" Style="{StaticResource CaptionTextBlockStyle}" IsTextSelectionEnabled="True"/>
+                        </StackPanel>
 
-            <StackPanel x:Name="HeaderPanel" Orientation="Horizontal">
-                <TextBlock x:Name="Header" x:Uid="FactoryOrchestrator" Grid.Column="0" Text="" TextWrapping="Wrap" VerticalAlignment="Center" HorizontalAlignment="Left" FontWeight="Bold" Style="{StaticResource TitleTextBlockStyle}"/>
-                <StackPanel x:Name = "MainControlPanel" Orientation="Horizontal" HorizontalAlignment="Center" Padding="40,0,0,0" Grid.Column="1" Grid.Row="0">
-                        <StackPanel x:Name="IndicatorPanel" Orientation="Horizontal">
-                            <StackPanel Orientation="Vertical" VerticalAlignment="Center" HorizontalAlignment="Left" Margin="5">
-                                <TextBlock x:Name="NetworkName" Text="" Style="{StaticResource CaptionTextBlockStyle}" VerticalAlignment="Center" TextWrapping="Wrap" IsTextSelectionEnabled="True"/>
-                                <TextBlock x:Name="NetworkIp" Text="" Style="{StaticResource CaptionTextBlockStyle}" VerticalAlignment="Center" TextWrapping="Wrap" IsTextSelectionEnabled="True"/>
```

Comment:
I had these visible and cycling on main page for a similar reason I had os version.

Devices might boot without working input, so being able to get IP address without having to do any user input was useful. I guess now DNS-SD helps you autodiscover devices running FO, which was not present before, but Id still consider keeping this on main page.

There should be ways to fix the size or area allowed to prevent UI shifing.

Reply:
The IP is already visible in the FO title. Are they not always the same value?

Reply:
if FO is running on the DUT, that just shows localhost. Many modern devices tend to have 4-5 IPs too, since they have a cell modem, wifi, BT etc.

Reply:
All current information is available on the home screen without clicking on a flyout now